python/dgl/distributed/rpc.py

```diff
@@ -458,8 +742,7 @@ def send_rpc_message(msg):
     ------
     ConnectionError if there is any problem with the connection.
     """
-    status = _CAPI_DGLRPCSendRPCMessage(msg)
-    # TODO: handle status flag
+    _CAPI_DGLRPCSendRPCMessage(msg)
```

Comment:
Status flag?

Reply:
what is the status flag?

Reply:
The status has been handled by C++ code.